modules/reference/pages/feature/Jakarta-EE9-feature-updates.adoc

```diff
@@ -0,0 +1,177 @@
+// Copyright (c) 2021 IBM Corporation and others.
+// Licensed under Creative Commons Attribution-NoDerivatives
+// 4.0 International (CC BY-ND 4.0)
+// https://creativecommons.org/licenses/by-nd/4.0/
+//
+// Contributors:
+// IBM Corporation
+//
+// This doc is hosted in the Red Hat Runtimes documentation. Any changes made to this doc also need to be made to the version that's located in the PurpleLiberty GitHub repo (https://github.com/PurpleLiberty/docs).
+//
+:page-description: For Jakarta EE 9 applications to run in Open Liberty, new Jakarta EE 9 features must be specified in the server configuration. If an old name is incorrectly used with a new Jakarta EE 9 version, the server does not start, and the server logs provide information to say which new feature name must be used.
+:seo-title: Jakarta EE9 feature updates - OpenLiberty.io
+:projectName: Open Liberty
+:page-layout: Jakarta EE9
+:page-type: feature updates
+= Jakarta EE9 feature updates
+
+For Jakarta EE 9 applications to run in Open Liberty, new Jakarta EE 9 features must be specified in the server configuration. If an old name is incorrectly used with a new Jakarta EE 9 version, the server does not start, and the server logs provide information to say which new feature name must be used.
+
+In many cases, the new Jakarta EE 9 features use the same short names as the corresponding Jakarta EE 8 features. In those cases, the Jakarta EE 9 features simply have an updated feature version.
+For example, `cdi-2.0` was updated to `cdi-3.0`.
+
+However, in some cases, the feature short name is also different, as with `ejb-3.2`, which was updated to `enterpriseBeans-4.0`.
+For example, if `ejb-3.2` is incorrectly changed to `ejb-4.0`, the server logs indicate that `enterpriseBeans-4.0` must be used.
+
+
+The following table lists features for which only the feature version is updated:
+
+.Jakarta EE 8 to Jakarta EE 9 feature updates: version only
```

Comment:
I think we we might want to add a column for the full names as well, and link them to the feature pages